junit-resource-poller/build.gradle

```diff
@@ -1,12 +1,13 @@
 apply from: "${rootDir}/gradle/publish.gradle"
 
 dependencies {
-    compile "com.squareup.okhttp3:okhttp"
-    compile "junit:junit"
-    compile "org.slf4j:slf4j-api"
+    compileOnly 'org.junit.jupiter:junit-jupiter-api'
```

Comment:
The idea here is that if you don't have junit5 stuff on your classpath already, then the junit-resource-poller won't bring it in, but the two new classes will not be usable.

If you do have it on your classpath, then `HttpPollingExtension` and `FailureCachingHttpPollingExtension` should work fine!